streetwhisperapp.py

```diff
@@ -115,27 +130,43 @@ def authorization():
 
     except KeyboardInterrupt:
         typer.Exit()
-    except:
+    except TypeError:
         invalidTokenPrompt = [
             {
                 'type': 'list',
                 'name': 'invalid_token',
-                'message': 'This token is invalid. Do you want to try again?',
+                'message': 'This token is invalid or there is an internal error (if you have seen this message previously).',
```

Comment:
I think we should put the email contact in case users repeatedly get this message (similar to how the message for the variable `internalErrorPrompt` is set up